src/server.ts

```diff
@@ -21,7 +21,7 @@ app.get("/", (req, res) => {
 });
 
 app.post("/score", async (req, res) => {
-  const { level, player } = req.body;
```

Comment:
Note removing player because we don't need it for scoring, we can do so indiscriminately without knowing origin, knowing it here actually can cause confusion as to who is supposed to persist this info - scoring service is dumb, dumb, dumb - just scores.